Rdmp.Dicom/CommandExecution/ExecuteCommandCFind.cs

```diff
@@ -30,7 +30,10 @@ public ExecuteCommandCFind(IBasicActivateItems activator, string start, string e
 
         var dir = Directory.CreateDirectory(outDir);
         var results = LoadDirectory.CreateDirectoryStructure(dir, "out", true);
```

Comment:
API would be cleaner if `CreateDirectoryStructure` could write the resulting paths to the `LoadMetaData` object instead of separately copying 3-4 new paths from the results object to the LMD object each time?

Reply:
That's a good idea, will add a ticket to add this in RDMP v8.2.1